vyperTypesGenerator.ts

```diff
@@ -1,47 +0,0 @@
-import { runTypeChain, glob } from 'typechain';
```

Comment:
why is it useless now?

Reply:
Ahh the types generator can add it back, but since this repo essentially only has solidity code I thought I'd remove it